### PR TITLE
Fix issues with Python 3 compatibility

### PR DESCRIPTION
**Pull request by vollbier**
I tried to fix all issues with Python 3 where a float was used as an index.
All examples in the Gallery compile now with Python 3.5

### CODE REVIEW

**Comment by jaidevd** [comment]
@vollbier Thanks for this! There are some issues with one testcase. They need to be modified a bit for Python 3.5.
Specifically, unittest in Python 3.5 doesn't have the `assertItemsEqual` method, needs to be replaced with `assertListEqual` or `assertTupleEqual`.
I could do the changes if you like. Can you grant me push access to your branch where I can make the changes?

**Comment by jaidevd** [comment]
Also, the `sphinx-apidoc` seems to have changed some behaviour. It now keeps the generated API reference docs in a folder named `doc/modules/generated` instead of the `_build` directory. The former needs to be gitignored.

File: tftb/processing/reassigned.py [hunk 1]
```diff
@@ -81,8 +81,8 @@ def pseudo_wigner_ville(signal, timestamps=None, n_fbins=None, fwindow=None):
         for jcol in range(n_fbins):
             if np.abs(tfr[jcol, icol]) > threshold:
                 jcolhat = jcol - tf2[jcol, icol]
-                jcolhat = np.remainder(np.remainder(jcolhat - 1,
-                                                    n_fbins) + n_fbins, n_fbins)
+                jcolhat =int(np.remainder(np.remainder(jcolhat - 1,
```
Comment (jaidevd) [inline tftb/processing/reassigned.py]: There needs to be a space between the `=` and `int`.

**Comment by vollbier** [comment]
@jaidevd I granted you access to vollbier/pytftb and added the gitignore file.

**Comment by jaidevd** [comment]
@vollbier Thanks, everything's fine now. Merging.

**Pull request merged**